renderer/src/features/mcp-servers/sub-pages/customize-tools/__tests__/page.test.tsx

```diff
@@ -0,0 +1,285 @@
+import { render, waitFor, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { describe, it, expect, beforeEach, afterEach, vi } from 'vitest'
+import { server as mswServer } from '@/common/mocks/node'
+import { http, HttpResponse } from 'msw'
+import { mswEndpoint } from '@/common/mocks/customHandlers'
+import type { CoreWorkload } from '@api/types.gen'
+import * as orchestrateRunLocalServer from '@/features/mcp-servers/lib/orchestrate-run-local-server'
+import * as orchestrateRunRemoteServer from '@/features/mcp-servers/lib/orchestrate-run-remote-server'
+import * as useUpdateServerModule from '@/features/mcp-servers/hooks/use-update-server'
+import {
+  createMemoryHistory,
+  createRootRoute,
+  createRoute,
+  Outlet,
+  Router,
+  RouterProvider,
+} from '@tanstack/react-router'
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query'
+import { CustomizeToolsPage } from '../page'
+
+const createLocalServerWorkload = (
```

Comment:
you can probably import an auto generated/customized fixture and just add the overrides here? (It's nit pick tho)

Reply:
Yeah, I’d take a look at what we have and extend the base scenario in the autogenerated code and just changing what we need